### PR TITLE
add nav-bar cart-icon hamburger-menu

### PR DESCRIPTION
**Pull request by gsaito2023**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -22,9 +26,19 @@ function App() {
 
   return (
     <div className='container-fluid'>
+       <Media query={{ maxWidth: 599 }}>
```
Comment (gsaito2023) [inline src/App.js]: Could you explain what is happening here? I don't understand it.

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.navbar-link:hover,
```
Comment (gsaito2023) [inline src/App.css:1]: Could you move your CSS changes to index.css so we consolidate where they are.

File: src/components/MdHeader.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                <img src="./logo.jpg" alt='logo' width='130px'></img>
+            </Link>
+            <button className="btn btn-danger me-3 rounded-pill fw-bold position-relative" type="button" data-bs-toggle="offcanvas" data-bs-target="#cart" aria-controls="cart">
+            <i className="bi bi-cart-fill me-1"></i>
+            <span className="position-absolute top-0 start-100 translate-middle badge rounded-pill bg-primary">
+                     0
+                <span className="visually-hidden">cart's items amount</span>
+            </span>
+            </button>   
+        <HeadOffCanvas/>
```
Comment (gsaito2023) [inline src/components/MdHeader.js:26]: Do you think there is enough spacing to tap on Sign In or Sign up links on a mobile device? They appear on the same line on hamburger menu.

**Review by wyzhaobvt — COMMENTED** [review]
review check

File: package.json [hunk 4]
```diff
@@ -3,11 +3,16 @@
   "version": "0.1.0",
```
Comment (gsaito2023) [inline package.json:3]: Testing comment

File: src/App.css (hunk 2 above)
Comment (wyzhaobvt) [reply src/App.css:1]: CSS changes of components in App.css

File: src/components/MdHeader.js (hunk 3 above)
Comment (wyzhaobvt) [reply src/components/MdHeader.js:26]: It is one button that trigger the sign in modal in default

File: src/App.js (hunk 1 above)
Comment (wyzhaobvt) [reply src/App.js]: it's the npm-react media package, so we can have conditional statement to have responsive navbar.

**Comment by gsaito2023** [comment]
Changes looks good to me.

**Pull request merged**